app/models/assessment.rb

```diff
@@ -0,0 +1,49 @@
+# coding: utf-8
+class Assessment < ApplicationRecord
+  include Swagger::Blocks
+
+  belongs_to :format
+  belongs_to :created_by, class_name: 'User'
+
+  enum visibility: %i[internal external]
+
+  has_many :questions
+
+  validates :format, :created_by, presence: true
+
+  swagger_schema :Assessment do
+    key :description, "The umbrella record for all things related to an exercises that a student could work, including its stem, answer, solutions, and variants"
+    key :required, [:id, :version, :visibility, :format_id, :content, :created_at]
+    property :id,           type: :string, format: :uuid
+    property :created_at,   type: :string, format: 'date-time'
+
+    property :identifier,
+             type: :string,
+             description: 'A unique value that can be used to link the assessment back to the contributors copy'
```

Comment:
This makes it sound unique in the contributor's scope, but the identifier is required to be globally unique within the mothership if I'm reading the AR below correctly.  If we want it globally unique within a15k but not make the contributing member deal with that, would we potentially prefix what we get with the member's ID?